Tests/SystemTests/Utils/RestHelper.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  File.swift
```

Comment:
In this and other new files there is this boilerplate. Please remove these. We can find out who created it from Git history and the boilerplate is misleading / adds cognitive load anyway for reviewers.

Reply:
f5eab85276b7643cc8eb8f5d791e4e24a74d37cf